src/macad_gym/envs/intersection/stop_sign_urban_intersection_3c.json

```diff
@@ -13,7 +15,8 @@
                 "send_measurements": false,
                 "enable_planner": true,
                 "spectator_loc": [140, 68, 9],
-                "synch_server": true
+                "sync_server": true,
+        	    "fixed_delta_seconds": 0.05
```

Comment:
Same. Needs a visual indent fix. Please remove the extra space in the beginning.